File: iosApp/iosApp/Presentation/Settings/ChangePassword/ChangePasswordUiState.swift
//
//  ChangePasswordUiState.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/15.
//

import Foundation

struct ChangePasswordUiState {
    var currentPassword: String
    var newPassword: String
    var confirmPassword: String

    static let empty = ChangePasswordUiState(
        currentPassword: "",
        newPassword: "",
        confirmPassword: ""
    )
}

File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/settings/changepassword/ChangePasswordUiState.kt
package com.hhp227.concafe.presentation.settings.changepassword

data class ChangePasswordUiState(
    val currentPassword: String,
    val newPassword: String,
    val confirmPassword: String,
    val isSubmitting: Boolean
) {
    companion object {
        fun empty(): ChangePasswordUiState = ChangePasswordUiState("", "", "", false)
    }
}
